gradle/generation/extract-jdk-apis.gradle

```diff
@@ -26,7 +26,7 @@ configure(rootProject) {
 
 configure(project(":lucene:core")) {
   ext {
-    apijars = file('src/generated/jdk');
+    apijars = layout.projectDirectory.dir("src/generated/jdk")
```

Comment:
can we define the `.file("jdk${jdkVersion}.apijar")` here as well instead of calling that twice below?

Reply:
Ah they already accepted and merged